src/Commands/InitCommand.php

```diff
@@ -269,6 +189,213 @@ public function handle(): void
         $this->runMigrations();
     }
 
+    protected function validateInput(callable $method, string $field, string|array $rules): string
```

Comment:
let's make refactoring of this method, it should get the parameter name as arg and validation rules, and generate prompt automatically

```suggestion
    protected function askWithValidation(string $parameter, string|array $rules): string
```